db/upgrade-X.X-X.Y.sql

```diff
@@ -119,7 +119,12 @@ ALTER TABLE pki_cas
 ALTER TABLE pki_revoked_certs
     ADD COLUMN IF NOT EXISTS `subject` varchar(255) AFTER crl_reason;
 
+\! echo "Grant binlog admin for pf user"
+GRANT BINLOG ADMIN ON *.* TO `pf`@`%`;
+GRANT BINLOG ADMIN ON *.* TO `pf`@`localhost`;
```

Comment:
This needs to be done via a script since the database name could be different.

Reply:
Actually because the username would be different since this adds the right to all DBs

Something like: addons/upgrade/to-10.0-upgrade-pf-privileges.sh